src/styles/main.scss

```diff
@@ -1 +1,69 @@
 // put code here
+.watch {
+  position: relative;
+  height: 500px;
+  width: 500px;
+  border-radius: 50%;
+  border: 10px dashed #000;
+
+
+ &__minute,
+ &__center,
+ &__second {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+ }
+
+ &__minute {
+  background-color: blue;
+  height: 125px;
+  width: 20px;
+
+  transform-origin: bottom;
+  transform: translate(-50%, -100%) rotate(0);
+
+  animation-name: min;
+  animation-duration: 20s;
+  animation-timing-function: linear;
+  animation-iteration-count: infinite;
+ }
+
+ &__second {
+  background-color: green;
+  height: 249px;
+  width: 10px;
+
+  transform-origin: bottom;
+  transform: translate(-50%, -100%) rotate(0);
+
+  animation-name: sec;
+  animation-duration: 10s;
+  animation-timing-function: linear;
+  animation-iteration-count: infinite;
```

Comment:
You now have the parameters for `transform` and `animation` repeated twice, only with a difference in `duration`. Try to use the features of SCSS to avoid repeating the code unnecessarily.